spotinst_sdk2/models/ocean_cd/strategy.py

```diff
@@ -0,0 +1,216 @@
+    """
+
+    def __init__(
+            self,
+            template_names: List[str] = none):
+        self.template_names = template_names
+
+
+class Scale:
+    """
+    # Arguments
+    match_traffic_weight: bool
+    replicas: int
+    weight: int
+    """
+
+    def __init__(
+            self,
+            match_traffic_weight: bool = none,
+            replicas: int = none,
+            weight: int = none):
+        self.match_traffic_weight = match_traffic_weight
+        self.replicas = replicas
+        self.weight = weight
+
+
+class Verification:
+    """
+    # Arguments
+    template_names: List[str]
+    """
+
+    def __init__(
+            self,
+            template_names: List[str] = none):
+        self.template_names = template_names
+
+
+class Pause:
+    """
+    # Arguments
+    duration: str
+    """
+
+    def __init__(
+            self,
+            duration: str = none):
+        self.duration = duration
+
+
+class HeaderValue:
+    """
+    # Arguments
+    exact: str
+    prefix: str
+    regex: str
+    """
+
+    def __init__(
+            self,
+            exact: str = none,
+            prefix: str = none,
+            regex: str = none):
+        self.exact = exact
+        self.prefix = prefix
+        self.regex = regex
+
+
+class Match:
+    """
+    # Arguments
+    header_name: str
+    header_value: HeaderValue
+    """
+
+    def __init__(
+            self,
+            header_name: str = none,
+            header_value: HeaderValue = none):
+        self.header_name = header_name
+        self.header_value = header_value
+
+
+class HeaderRoute:
+    """
+    # Arguments
+    name: str
+    match: List[Match]
+    """
+
+    def __init__(
+            self,
+            name: str = none,
+            match: List[Match] = none):
+        self.name = name
+        self.match = match
+
+
+class Steps:
+    """
+    # Arguments
+    name: str
+    set_weight: int
+    set_canary_scale: Scale
+    verification: Verification
+    pause: Pause
+    set_header_route: HeaderRoute
+    """
+
+    def __init__(
+            self,
+            name: str = none,
+            set_weight: int = none,
+            set_canary_scale: Scale = none,
+            verification: Verification = none,
+            pause: Pause = none,
+            set_header_route: HeaderRoute = none):
+        self.name = name
+        self.set_weight = set_weight
+        self.set_canary_scale = set_canary_scale
+        self.verification = verification
+        self.pause = pause
+        self.set_header_route = set_header_route
+
+
+class Canary:
+    """
+    # Arguments
+    background_verification: BackgroundVerification
+    steps: List[Steps]
+    """
+
+    def __init__(
+            self,
+            background_verification: BackgroundVerification = none,
+            steps: List[Steps] = none):
+        self.background_verification = background_verification
+        self.steps = steps
+
+
+class RollingPause:
```

Comment:
Previous 'Pause' class can be used instead of redefining.

Reply:
Done